test/lib_ic/test_ic_spec/test_ic_spec.py

```diff
@@ -213,14 +215,22 @@ def get_suppression_arr(input_audio, output_audio):
     for i in np.arange(0, num_frames*frame_advance, frame_advance):
         i = int(i)
         in_rms = rms(input_audio[0][i:i+frame_advance])
-        out_rms = rms(output_audio[0][i:i+frame_advance])
```

Comment:
You could do something like, out_rms = out_rms.reshape(1 , out_rms.shape[-1]) to force it into a 2D shape. Then you'll not need the if condition below.

Reply:
The output_audio can either be a single channel or have two channels. This will work fine for a single-channel array. However, I can't reshape a (2, 240) array into a (1, 240) array, python will give me an error. So I'll have to have an if ether way

Reply:
Right, that makes sense. I thought the output was either shaped (n,) or (1,n).